src/scenes/Users/List/UserList.tsx

```diff
@@ -0,0 +1,56 @@
+import { Grid, makeStyles, Typography } from '@material-ui/core';
+import React, { FC } from 'react';
+import { User } from '../../../api';
+import { ListContainer } from '../../../components/ListContainer';
+import { SortButtonDialog, useSort } from '../../../components/Sort';
+import { UserListItemCard } from '../../../components/UserListItemCard';
+import { listOrPlaceholders } from '../../../util';
+import { useUsersQuery } from './users.generated';
+import { UserSortOptions } from './UserSortOptions';
+
+const useStyles = makeStyles(({ spacing }) => ({
+  options: {
+    margin: spacing(3, 0),
+  },
+  projectItem: {
+    marginBottom: spacing(2),
+  },
+}));
+
+export const UserList: FC = () => {
+  const sort = useSort<User>();
+
+  const { data } = useUsersQuery({
+    variables: {
+      input: {
+        ...sort.value,
+      },
+    },
+  });
+  const classes = useStyles();
+
+  return (
+    <ListContainer>
+      <Typography variant="h2" paragraph>
+        People
+      </Typography>
+      <Grid container spacing={1} className={classes.options}>
+        <Grid item>
+          <SortButtonDialog {...sort}>
+            <UserSortOptions />
+          </SortButtonDialog>
+        </Grid>
+      </Grid>
+      <Typography variant="h3" paragraph>
+        {data?.users.total} People
+      </Typography>
+      <Grid container spacing={3}>
+        {listOrPlaceholders(data?.users.items, 5).map((item, index) => (
+          <Grid item key={item?.id ?? index} xs={12} md={6} lg={4} xl={3}>
```

Comment:
There seemed to be a conflict with the Grid and flexbox. Chrome was adding more spacing than desired - I think between loading skeletons and actual data. I think I moved away from this with project list (d6f896e6713d532aebd6e3fcf107f4c4d28a0ab0) and organization list (54ef429719a5cd4ae774aac41d42e6cc55199058).

Do you notice that weirdness as well?

Reply:
I'm not seeing any difference in width between the grid in loading state and the grid with data... but maybe I'm not looking for the right thing? I felt like some sort of grid display was needed here because these cards are so much narrower than the Projects/Languages/Partners cards. But I notice that there's a second People Card UI depicted in the Figma design that's landscape orientation; do you want to go that direction instead?

Reply:
Yeah I would've gone with the landscape one at first too. Esp since most of the emphasis is on the picture which we don't have yet. You to you, either way I don't think this list scene is a big priority.